plugin.video.sandmann/addon.py

```diff
@@ -16,25 +16,53 @@
 #
 
 import sys
+import threading
 
 from sandmann import getEpisodeData
 
-# import xbmc
+import xbmc
 import xbmcplugin
 import xbmcgui
 import xbmcaddon
 
 episodes_url = "https://appdata.ardmediathek.de/appdata/servlet/tv/Sendung?documentId=6503982&json"
 
 addon = xbmcaddon.Addon()
-quality = int(addon.getSetting("quality"))
+# addon_name = addon.getAddonInfo('name')
+# addon_icon = addon.getAddonInfo('icon')
 
-title, thumbnail_image, fanart_image, stream = getEpisodeData(
+addon_handle = int(sys.argv[1])
+base_path = sys.argv[0]
+
+quality = addon.getSettingInt("quality")
+update = addon.getSettingInt("update")
+interval = addon.getSettingInt("interval")
+
+title, description, thumbnail_image, fanart_image, stream = getEpisodeData(
     episodes_url, quality)
 
-li = xbmcgui.ListItem(label=title, thumbnailImage=thumbnail_image)
-li.setProperty("fanart_image", fanart_image)
+li_episode = xbmcgui.ListItem(label=title, thumbnailImage=thumbnail_image)
```

Comment:
Please remove thumbnailImage kwarg and use the setArt method on the ListItem object:

```python
li_episode.setArt({'thumb':thumbnail_image, 'fanart':fanart_image})
```

Reply:
okay, I'll do this in the next version.